types/mini-css-extract-plugin/index.d.ts

```diff
@@ -97,6 +103,12 @@ declare namespace MiniCssExtractPlugin {
          * @default true
          */
         esModule?: boolean;
+
+        /**
+         * Layer of the css execution
+         */
+        layer?: string;
```

Comment:
where are you getting this from? the sources don't specify a layers option anywhere. i suppose this PR isn't from experiencing a missing type, but is from reading the README? as they do mention layers, but meaning webpack's own `experiments.layers`, not a loader option

Reply:
aha it is hidden away in the readme:

> When combined with `experiments.layers`, this adds a `layer` option to the loader options to specify the layer of the css execution.

so its used only when `layers` is enabled, they don't currently have it in their list of loader options so was easy to miss